moveit_ros/moveit_servo/src/collision_check.cpp

```diff
@@ -201,33 +209,40 @@ void CollisionCheck::run(const ros::TimerEvent& timer_event)
 
   // publish message
   {
-    auto msg = moveit::util::make_shared_from_pool<std_msgs::Float64>();
-    msg->data = velocity_scale_;
-    collision_velocity_scale_pub_.publish(msg);
+    auto msg = std::make_unique<std_msgs::msg::Float64>();  // TODO(adamp): consider going back to pool allocate
```

Comment:
I think you could delete these pool TODOs. Create an issue for later

Reply:
Done, I will make that issue